### PR TITLE
✨ Add Traefik v3 instructions

### PR DESCRIPTION
**Pull request by rbauduin**
The only difference is how swarm mode is activated, replacing
```
--providers.docker.swarmmode
```
by
```
--providers.swarm.endpoint=unix:///var/run/docker.sock
```

### CODE REVIEW

**Comment by ewurch** [comment]
Hi @rbauduin, I'm a fellow user of this stack for my VPS and I really liked your initiative to update Traefik. But I think you forgot to update the image tag on the traefik-v3 file:
 [https://github.com/tiangolo/dockerswarm.rocks/pull/93/files#diff-eed813aa598624c5d6849a2de675909068d9c0a5592a6db3bc6810fcd06a1001R7](https://github.com/tiangolo/dockerswarm.rocks/pull/93/files#diff-eed813aa598624c5d6849a2de675909068d9c0a5592a6db3bc6810fcd06a1001R7)

File: docs/traefik-v3.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+version: '3.3'
+
+services:
+
+  traefik:
+    # Use the latest v2.2.x Traefik image available
+    image: traefik:v2.2
```
Comment (ewurch) [inline docs/traefik-v3.yml:7]: Shouldn't it supposed to be v3?

Comment (Olegt0rr) [reply docs/traefik-v3.yml:7]: v3 still in beta

Comment (rbauduin) [reply docs/traefik-v3.yml]: yes, good catch, pushing an update

**Comment by tiangolo** [comment]
Thank you! 🍰 ☕

**Pull request merged**